psychopy/visual/movie3.py

```diff
@@ -312,7 +312,13 @@ def _updateFrameTexture(self):
             if self._nextFrameT > (self._videoClock.getTime() -
                                    self._retraceInterval/2.0):
                 return None
-        self._numpyFrame = self._mov.get_frame(self._nextFrameT)
+        try:
+            self._numpyFrame = self._mov.get_frame(self._nextFrameT) 
+        except:
+            if self.autoLog:
+                self.win.logOnFlip("Frame %s not found, moving one frame & trying again" % (str(self._nextFrameT)), level=logging.WARNING, obj=self)
```

Comment:
A warning message like this doesn't need to be time-stamped with the screen refresh. Easier just to issue a warning immediately:
```
logging.warning("Frame {} not found, moving one frame and trying again" 
                .format(self._nextFrameT), obj=self)
```
I also tend to use `{}` and `format(entries)` for string formatting these days - it makes for neater auto-conversion of entries